blocklock-agent/Cargo.toml

```diff
@@ -22,6 +22,7 @@ serde = { workspace = true, features = ["derive"] }
 
 # misc
 anyhow = { workspace = true }
+serde_keys = { version = "0.1.0", path = "../crates/serde-keys" }
```

Comment:
we should probably register serde_keys in the root `Cargo.toml` and change that to `serde_keys.workspace = true`